### DISCUSSION TITLE
Is there a way to automatically remove unused imports?

### DISCUSSION THREAD

**Discussion by 11Firefox11** [biomejs/biome#304] [Q&A] [answered]
I browsed through the docs and searched a few in the issues, but didn't find any reference to this. Can you automatically remove unused imports with Biome?

**Comment by Conaclos** [comment]
Not for now. We could add a suggestion for removing the import in `noUnusedVariables`. We could also automatically remove unused imports when imports are sorted. What do you think?

**Reply by 11Firefox11** [reply]
Automatically removing unused imports sounds great. That feature would be useful me. In my projects most of the time I auto import things and later on I forget to remove them.

**Reply by 11Firefox11** [reply]
I would be grateful for that feature.
Should I create a new idea discussion about this? (As I see I can't create a feature request issue.)

**Reply by Conaclos** [reply]
> Should I create a new idea discussion about this? (As I see I can't create a feature request issue.)

You can. This could make the idea more visible.

**Reply by 11Firefox11** [reply, marked as answer]
New discussion: #305 .
